streams/streams-scala/src/main/scala/org/apache/kafka/streams/scala/serialization/Serdes.scala

```diff
@@ -43,12 +43,6 @@ object Serdes extends LowPrioritySerdes {
   implicit def javaIntegerSerde: Serde[java.lang.Integer] = JSerdes.Integer()
   implicit def uuidSerde: Serde[UUID] = JSerdes.UUID()
 
-  implicit def timeWindowedSerde[T](implicit tSerde: Serde[T]): WindowedSerdes.TimeWindowedSerde[T] =
-    new WindowedSerdes.TimeWindowedSerde[T](tSerde)
-
-  implicit def sessionWindowedSerde[T](implicit tSerde: Serde[T]): WindowedSerdes.SessionWindowedSerde[T] =
-    new WindowedSerdes.SessionWindowedSerde[T](tSerde)
```

Comment:
Ah, we don't technically need to drop this one, only the `timeWindowedSerde`. Sorry for the confusion.

Reply:
Adding it back will also unblock the build, which fails right now because `WindowedSerdes` is an unused import. I'll just add this back and push to your branch, assuming the rest of the tests pass, so I can go ahead and merge it.

Reply:
Ok, thanks!